src/webots/gui/WbWrenWindow.cpp

```diff
@@ -69,8 +69,10 @@ WbWrenWindow::WbWrenWindow() :
 
   setSurfaceType(QWindow::OpenGLSurface);
 
+  const WbVersion openGLTargetVersion(3, 3);
```

Comment:
That's just an idea but maybe because the target version is imposed by WREN we should encode it on the WREN side.

Reply:
It seems that neither WREN nor GLAD are currently linked to a specific OpenGL version. We only check at the WREN initialization that some OpenGL extensions are available. But Webots/Qt set OpenGL 3.3, because they manage the windows, the OpenGL initialization and the shaders. Shaders are the (only?) main reason we need OpenGL 3.3.

It has some sense, and this PR is correct according to this state.

If we would like to change this, it's not related with this PR and would require a new GitHub issue and a stronger refactor.

Reply:
Correction: glad.h is actually linked with an OpenGL version when generating glad.h